RiotSharp/LeagueEndpoint/LeaguePosition.cs

```diff
@@ -5,38 +5,60 @@ namespace RiotSharp.LeagueEndpoint
     /// <summary>
     /// Team or summoner in a league (League API).
     /// </summary>
-    public class LeagueEntry
+    public class LeaguePosition
     {
-        internal LeagueEntry() { }
+        internal LeaguePosition() { }
 
         /// <summary>
+        /// The name of the league of the participant.
+        /// Only when it's called from the GetLeaguePositions()
+        /// </summary>
+        [JsonProperty("leagueName")]
+        public string LeagueName { get; set; }
+
+        /// <summary>
+        /// The queue type of the league.
+        /// Only for the GetLeaguePositions() -> don't exist when it's an entry from a League
+        /// as there is already the Queue property in this case.
+        /// </summary>
+        [JsonProperty("queueType")]
+        public string QueueType { get; set; }
+        
+        /// <summary>
         /// The league division of the participant.
         /// </summary>
-        [JsonProperty("division")]
+        [JsonProperty("rank")]
         public string Division { get; set; }
 
+        ///<summary>
+        /// The league tier of the participant.
+        /// Only when it's called from the GetLeaguePositions()
+        /// </summary>
+        [JsonProperty("tier")]
+        public string Tier { get; set; }
+
         /// <summary>
         /// Specifies if the participant is fresh blood.
         /// </summary>
-        [JsonProperty("isFreshBlood")]
+        [JsonProperty("freshBlood")]
         public bool IsFreshBlood { get; set; }
```

Comment:
Let's rename the properties too, valid for the fields below too

Reply:
Okay, I was keeping the is because it's common when using a bool to have it nammed "isSomething" but alright, will changed that soon.

Reply:
We generally to match what Riot is sending as much as possible :+1: